src/pkg/cloudclient/restapi/client.go

```diff
@@ -67,8 +67,12 @@ type doerWithErrorCheck struct {
 	doer Doer
 }
 
+type ResponseSingleError struct {
+	Message string `json:"message"`
+}
+
 type ResponseBody struct {
-	Message string `json:"message,omitempty"`
+	Errors []ResponseSingleError `json:"errors"`
```

Comment:
You can use an anonymous struct here to make this simpler (no need to actually define ResponseSingleError)
```
type ResponseBody struct {
	Errors []struct {
		Message string `json:"message"`
	} `json:"errors"`
}```